cirq/contrib/contrib-requirements.txt

```diff
@@ -1,4 +1,5 @@
 # Runtime requirements for contrib.
 
+ply==3.4
```

Comment:
@Strilanc - what is the recommendation here? Based on the google3 availability, I pinned it to 3.4. However qiskit uses the latest and breaks with 3.4 which is old, so users who have qiskit installed will have some issues if they run `pip install -e cirq/contrib/contrib-requirements.txt`

Reply:
CCing @dstrain115  on this.

Reply:
You may have to contact the person responsible for the google3 import and get them to sync.

Reply:
We talked this through with @dstrain115, ply 3.4 should be fine as a dependency. 
I came to the conclusion that `ply>=3.4` might be the right way to go, if we don't want to break people who want to install cirq into an env that already has qiskit installed. 
However, the QASM importer should be tested against 3.4 in Travis to make sure it works internally too. So I would leave this as `ply ==3.4`, unless users ask for loosening the range.